jsk_fetch_robot/fetcheus/fetch-interface.l

```diff
@@ -130,9 +130,15 @@
       (setq result (send gripper-action :get-result))
       result))
   ;;
-  (:speak (text &rest args)
-    (let ()
-      (apply #'speak-en text :topic-name "/sound_play" args)))
+  (:speak-en (text &rest args)
```

Comment:
is this common interface with pr2 robot or robot-interface?
Cc: @snozawa

Reply:
@k-okada This is not common now but uses the same functions as PR2. AFAIK, this style is first emerged in `naoqieus` https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_naoqi_robot/naoqieus/naoqi-interface.l#L250 .